packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1192,16 +1194,22 @@ export const VIDEO_RESOLUTIONS = {
 };
 
 /**
- * Max frame sizes based on h264 configs
+ * Max frame sizes and bandwith constraints based on h264 configs
  * https://en.wikipedia.org/wiki/Advanced_Video_Coding
  */
-export const MAX_FRAMESIZES = {
-  [QUALITY_LEVELS.LOW]: 1620,
-  [QUALITY_LEVELS.MEDIUM]: 3600,
-  [QUALITY_LEVELS.HIGH]: 8192
+export const REMOTE_VIDEO_CONSTRAINTS = {
+  MAX_FS: {
+    [QUALITY_LEVELS.LOW]: 1620,
+    [QUALITY_LEVELS.MEDIUM]: 3600,
+    [QUALITY_LEVELS.HIGH]: 8192
+  },
+  MAX_BITRATE: {
```

Comment:
the name is wrong - this is not bitrate, "max-mbps" is the decoder's maximum processing speed in macroblocks per second